pretyping/arguments_renaming.ml

```diff
@@ -69,19 +69,25 @@ let rename_arguments local r names =
 
 let arguments_names r = GlobRef.Map.find r !name_table
 
-let rec rename_prod c = function 
+let get_rename old_name rename_item =
+  match rename_item with
+  | Name _ as x -> x
+  | Anonymous -> old_name
+
+let rec rename_type_aux c = function
```

Comment:
This is probably missing a comment explaining the intended behavior. Given the name, shouldn't it be local to `rename_type`?

Reply:
done